#!/usr/bin/env python
PACKAGE = "core"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("qualifier_wait", int_t, 0, "Initial wait time", 15, 0, 60)
gen.add("qualifier_depth", double_t, 0, "Depth", 0.4, 0, 4)
gen.add("qualifier_forward", double_t, 0, "Forward", 12, 0, 20)

gen.add("sauvc_depth", double_t, 0, "depth sauvc", 0.3, 0, 4)
gen.add("sauvc_forward", double_t, 0, "forward", 2, 0, 5)
gen.add("linefollower_depth", double_t, 0, "bucket depth", 0.3, 0, 4)
gen.add("bucket_depth", double_t, 0, "bucket depth", 0.2, 0, 4)
#gen.add("qualifier_lane_length", int_t, 0, "The length of the qualifier", 10, 0, 20)
#gen.add("sauvc_depth_normal", double_t, 0, "Depth setpoint", 0.4, 0, 4)
exit(gen.generate(PACKAGE, "mission_planner", "mission_planner"))
